### DISCUSSION TITLE
Custom SystemAccentColor not visible when overriding in FluentTheme

### DISCUSSION THREAD

**Discussion by MichaelSvejcar** [AvaloniaUI/Avalonia#11205] [Q&A]
I'm using Avalonia UI with the FluentTheme, and I'm trying to override the **SystemAccentColor** for my application to change the accent color for all controls. I've overridden the **SystemAccentColor** in my **App.axaml** file, but now there is no color at all and all the parts of controls where one would expect the newly defined color, are now just transparent.

Here's my **App.axaml**:
```
<Application xmlns="https://github.com/avaloniaui"
             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
             xmlns:local="using:MyAvaloniaApp"
             xmlns:views="using:MyAvaloniaApp.Views"
             x:Class="MyAvaloniaApp.App">
    <Application.DataTemplates>
        <local:ViewLocator/>
    </Application.DataTemplates>
    <Application.Resources>
        <SolidColorBrush x:Key="SystemAccentColor" Color="#CCFF0068"/>
    </Application.Resources>
    <Application.Styles>
        <FluentTheme Mode="Dark"/>
        <StyleInclude Source="avares://Avalonia.Controls.DataGrid/Themes/Default.xaml"/>
        <StyleInclude Source="/Views/Templates/SomeWindow.axaml"/>
    </Application.Styles>
</Application>
```

What could be causing this issue, and how can I correctly override the **SystemAccentColor** for all controls?

**Comment by maxkatz6** [comment]
> and how can I correctly override the SystemAccentColor for all controls?

See new API https://github.com/AvaloniaUI/Avalonia/pull/11097#issue-1679291634 in preview7
